build-system/tasks/check-types.js

```diff
@@ -122,7 +166,10 @@ const TYPE_CHECK_TARGETS = {
   // introduced. It is okay to remove a file from this list only when fixing a
   // bug for cherry-pick.
   'pride': {
-    srcGlobs: PRIDE_FILES_GLOBS,
+    srcGlobs: [
+      ...PRIDE_FILES_GLOBS,
+      ...globby.sync('src/core/**/*.js').filter((p) => !p.includes('extern')),
```

Comment:
@rcebulko: do you have a better idea for this line?

Reply:
https://github.com/ampproject/amphtml/pull/33957#discussion_r623152190